components/brave_rewards/resources/brave_rewards_static_resources.grd

```diff
@@ -9,6 +9,7 @@
   <release seq="1">
     <includes>
       <!-- WebUI Brave Rewards Page resources -->
+      <include name="IDR_BRAVE_REWARDS_ANDROID_PAGE_HTML" file="android_page/brave_rewards_page.html" type="BINDATA" />
       <include name="IDR_BRAVE_REWARDS_PAGE_HTML" file="page/brave_rewards_page.html" type="BINDATA" />
       <include name="IDR_BRAVE_REWARDS_IMG_WALLET_BG" file="page/img/wallet_bg.svg" type="BINDATA" />
       <include name="IDR_BRAVE_REWARDS_IMG_CAPTCHA_BAT" file="page/img/captcha_bat.png" type="BINDATA" />
```

Comment:
this is not used afaik

Reply:
It's used here, we conditionally load the Android-specific page when running on Android:

https://github.com/brave/brave-core/pull/3886/files#diff-472c4e44690bc2b99a1d2d0905870c3cR1690

Reply:
nvm, I missed that this is general one as I was not sure why we would need captcha on android